examples/apps/src/ble_advertise.rs

```diff
@@ -19,7 +20,7 @@ where
     info!("Our address = {:?}", address);
 
     let mut resources: HostResources<DefaultPacketPool, 0, 0> = HostResources::new();
-    let stack = trouble_host::new(controller, &mut resources).set_random_address(address);
+    let stack = trouble_host::new(controller, &mut resources, IoCapabilities::NoInputNoOutput).set_random_address(address);
```

Comment:
This is a breaking change of current API. 

Maybe

```rust
trouble_host::new(controller, &mut resources).set_random_address(address).set_io_capabilities(IoCapabilities::NoInputNoOutput);
```

is better?

Reply:
Done